graphics/rendering/blender/blender.def

```diff
@@ -1,9 +1,19 @@
-Bootstrap: library
-From: ubuntu:16.04
+Bootstrap: docker
```

Comment:
Why did you change this to docker?

Reply:
A few reasons:

1. Our bootstrapped library ubuntu image generally needs updating. A full `apt-get update && apt-get -y upgrade` takes time to run
2. The library image does not have the full set of Ubuntu apt sources setup. This means they need to be added manually, or `software-properties-common` must be installed to use `apt-add-repository`. That confuses the definition file and makes building longer.
3. The library `18.04` is base on `18.04.2` so a lot of updated `18.04.3` packages have to come in for blender from the ppa.

These things need to be addressed with the library image, but I don't have time at present.